mmdeploy/backend/ipu/wrapper.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Copyright (c) 2022 Graphcore Ltd. All rights reserved.
+from typing import Any, Dict, Optional, Sequence, Union
+
+import torch
+import onnx
+
+from mmdeploy.utils import Backend
+from mmdeploy.utils.timer import TimeCounter
+from ..base import BACKEND_WRAPPER, BaseWrapper
+
+import argparse
+from datetime import timedelta
+import numpy as np
+import model_runtime
+import popef
+import time
+
+
+@BACKEND_WRAPPER.register_module(Backend.IPU.value)
+class IPUWrapper(BaseWrapper):
+    """IPU engine wrapper for inference.
+
+    Args:
+        ONNX : onnx file for engine to wrap.
+        output_names (Sequence[str] | None): Names of model outputs  in order.
+            Defaults to `None` and the wrapper will load the output names from
+            model.
+
+    Note:
+        If the engine is converted from onnx model. The input_names and
+        output_names should be the same as onnx model.
+
+    Examples:
+        >>> from mmdeploy.backend.ipu import IPUWrapper
+        >>> popef_file = 'resnet.popef'
+        >>> model = IPUWrapper(popef_file)
+        >>> inputs = dict(input=torch.randn(1, 3, 224, 224))
+        >>> outputs = model(inputs)
+        >>> print(outputs)
+    """
+
+    def __init__(self,
+                 popef_file: str,
+                 bps=1,
+                 output_names: Optional[Sequence[str]] = None):
+        super().__init__(output_names)
+        print('init ipu backend with bps ', bps,
```

Comment:
Please use mmdeploy.utils.get_root_logger instead of print

Reply:
ok